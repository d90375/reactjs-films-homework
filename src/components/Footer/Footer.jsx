import React from "react";
import "./Footer.scss";

const Footer = () => {
  return (
    <div className="footer">
      <div className="footer__title">
        <span>FILMS</span>
      </div>
      <div className="footer__copyright">
        <span>Copyright Â© 2018 d90375</span>
      </div>
    </div>
  );
};

export default Footer;
