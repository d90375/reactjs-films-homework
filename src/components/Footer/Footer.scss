.footer {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  height: 263px;
  box-shadow: 0 10px 40px rgba(0, 0, 0, 0.04);
  background-color: #fff;
}

.footer__title {
  margin-top: 75px;
  color: #0aaee4;
  font-family: "Helvetica Neue", serif;
  font-size: 50px;
  font-weight: 400;
  text-transform: uppercase;
  letter-spacing: 8px;
}

.footer__copyright {
  margin-top: 75px;
  opacity: 0.4;
  color: #242b2d;
  font-family: "Helvetica Neue", serif;
  font-size: 24px;
  font-weight: 400;
  letter-spacing: 1.2px;
}
