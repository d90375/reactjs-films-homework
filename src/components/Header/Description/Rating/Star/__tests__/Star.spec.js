import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Footer from "../Star";

const renderer = new ShallowRenderer();
const setUp = (props) => {
  return renderer.render(<Footer {...props} />);
};

describe("Star component", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });

  it("should render Star component", () => {
    expect(component).toMatchSnapshot();
  });
});
