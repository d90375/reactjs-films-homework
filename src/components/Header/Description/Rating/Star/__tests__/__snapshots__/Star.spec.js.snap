// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star component should render Star component 1`] = `
<div
  className="star"
>
  <svg
    height="36"
    viewBox="0 0 36 36"
    width="36"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <g>
        <path
          d="M36 13.5H22.22L18.002 0l-4.22 13.5H0l11.165 8.374L6.75 36 18 27l11.25 9-4.414-14.127z"
          fill="#0aaee4"
        />
      </g>
    </g>
  </svg>
</div>
`;
